plugin/trino-hive/src/test/java/io/trino/plugin/hive/HiveQueryRunner.java

```diff
@@ -248,6 +248,7 @@ public DistributedQueryRunner build()
                         .put("hive.max-split-size", "10kB") // so that each bucket has multiple splits
                         .put("hive.storage-format", "TEXTFILE") // so that there's no minimum split size for the file
                         .put("hive.compression-codec", "NONE") // so that the file is splittable
+                        .put("hive.file-listing-timeout", "2.00m")
```

Comment:
maybe add a comment here on what it does

Reply:
yes. is this a test? I'm not sure if it's supposed to be in the UML diagram, but since it's in the test folder I won't add it ig.

Reply:
This is not a test. Its the querry runner (server) for the hive plugin.

Reply:
...but the file is in the test folder, so it's not a part of the main project?

Reply:
Oh you are right! @cadye Intressting. I guess it does not need to be part of the UML diagram. But the HiveConfig class probably should. But is this an issue blocking the merge ?

Reply:
HiveConfig has already been added to the UML. I don't see how changing in the test file would block the merge, but I haven't started merging yet.